crates/sui-replay/src/lib.rs

```diff
@@ -359,7 +356,7 @@ pub async fn execute_replay_command(
                 println!("{:#?}", sandbox_state.pre_exec_diag);
             }
             if show_effects {
-                println!("{:#?}", sandbox_state.local_exec_effects);
+                println!("{}", sandbox_state.local_exec_effects.to_string());
```

Comment:
The call to `to_string` should be unnecessary here.
```suggestion
                println!("{}", sandbox_state.local_exec_effects);
```